backend-api/src/functions/tests/pact/createApp.ts

```diff
@@ -12,7 +14,18 @@ export async function createApp(): Promise<Application> {
     const result = await asyncTokenHandlerConstructor({
       headers: req.headers,
       body: req.body,
-      dependencies: stateConfig.asyncTokenDependencies,
+      dependencies: asyncTokenStateConfig.dependencies,
+    });
+
+    res.status(result.statusCode);
+    res.send(JSON.parse(result.body));
```

Comment:
What's the rationale for this?

Reply:
I assume if anything we'd need to stringify it?

Reply:
As an initial thought, it looks like it needs to be parsed so it can be asserted against in the test

Reply:
> As an initial thought, it looks like it needs to be parsed so it can be asserted against in the test

This is the case